applications/ConvectionDiffusionApplication/python_scripts/coupled_structural_thermal_solver.py

```diff
@@ -7,6 +7,7 @@
 # Import applications
 import KratosMultiphysics.StructuralMechanicsApplication as KratosSMA
 import KratosMultiphysics.ConvectionDiffusionApplication as ConvDiff
+import KratosMultiphysics.ConstitutiveLawsApplication    as KratosCLApp
```

Comment:
is this always necessary?
Cannot it be used without too?

Reply:
sorry it's a leftover, rrrremoving it :D